File: frontend/src/app/models/reviewInfo.ts
export interface ReviewInfo {
    id: number;
    rating: number;
    comment: string;
    dateReviewed: Date;
    foodName: string;
    foodPhotoUrl: string;
    foodAverageRating: number;
    appUserName: string;
    appUserPhotoUrl: string;
}

File: backend/API/DTOs/ReviewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using API.Entities;

namespace API.DTOs
{
    public class ReviewInfo
    {
        public int Id { get; set; }
        public string Comment { get; set; }
        public string Rating { get; set; }
        public DateTime DateReviewed { get; set; }
        public string FoodName { get; set; }
        public string FoodPhotoUrl { get; set; }
        public double FoodAverageRating { get; set; }
        public int FoodId { get; set; }
        public string AppUserName { get; set; }
        public string AppUserPhotoUrl { get; set; }
    }
}